scripts/Makefile.lib

```diff
@@ -265,7 +265,9 @@ cmd_gzip = (cat $(filter-out FORCE,$^) | gzip -n -f -9 > $@) || \
 # ---------------------------------------------------------------------------
 
 DTC_OVERLAY_DIR ?= $(PROJECT_BASE)
-DTC_OVERLAY_FILE = $(abspath $(DTC_OVERLAY_DIR)/$(BOARD_NAME).overlay)
```

Comment:
@agross-linaro @d3zd3z I also prefer the overlay path being added as -I to the compiler